hathor/merged_mining/bitcoin.py

```diff
@@ -923,23 +923,25 @@ def create_output_script(address: bytes) -> bytes:
     # Prefixes: https://en.bitcoin.it/wiki/List_of_address_prefixes
     if address[0] in {0x00, 0x6f}:  # Base58 address starts with 1.. (mainnet) or m/n.. (testnet)
         assert len(address) == 25
-        # Pay To Public Key Hash (P2PKH): OP_DUP OP_HASH160 <PubKeyHash> OP_EQUALVERIFY OP_CHECKSIG
-        # See: https://bitcoin.org/en/transactions-guide#pay-to-public-key-hash-p2pkh
+        # P2PKH: OP_DUP OP_HASH160 OP_PUSHBYTES_20 <PubKeyHash> OP_EQUALVERIFY OP_CHECKSIG
+        # See: https://developer.bitcoin.org/devguide/transactions.html#pay-to-public-key-hash-p2pkh
```

Comment:
It's odd that even though the `OP_PUSHBYTES_20` is needed, the reference link does not mention it.

Reply:
Yes. That's what made me miss it. I should've checked by looking at transactions already on the blockchain though.